test/unit/controllers/declaration/CusCodeControllerSpec.scala

```diff
@@ -123,15 +123,12 @@ class CusCodeControllerSpec extends ControllerSpec {
 
     onClearance { declaration =>
       "return 303 (SEE_OTHER)" when {
-        "accept submission and redirect" in {
-          withNewCaching(declaration)
-          val correctForm = formData("12345678")
+        "invalid journey CLEARANCE" in {
+          withNewCaching(aDeclaration(withType(DeclarationType.CLEARANCE)))
 
-          val result = controller.submitForm(Mode.Normal, itemId)(postRequest(correctForm))
+          val result = controller.displayPage(Mode.Normal, "").apply(getRequest())
 
-          await(result) mustBe aRedirectToTheNextPage
-          thePageNavigatedTo mustBe controllers.declaration.routes.NactCodeController.displayPage(Mode.Normal, itemId)
-          verify(mockPage, times(0)).apply(any(), any(), any())(any(), any())
+          status(result) mustBe SEE_OTHER
```

Comment:
Can we check that we've been re-directed to the start page?  I think there are other examples of testing this.